### PR TITLE
MAISTRA-894: Updates for 1.1.0

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: Dockerfile.citadel [hunk 1]
```diff
@@ -13,9 +13,9 @@ LABEL maintainer="Istio Feedback <istio-feedback@redhat.com>"
 ENV container="oci"
 ENV ISTIO_VERSION=1.4.6
 
-RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/istio.el8/repo/rhelbeta-8/group_maistra-istio.el8-rhelbeta-8.repo -o /etc/yum.repos.d/openshift-istio-istio-epel-8.repo && \
+RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/istio-1.1/repo/epel-8/group_maistra-istio-1.1-epel-8.repo -o /etc/yum.repos.d/maistra-1.1-epel-8.repo && \
     microdnf update -y && \
-    microdnf install -y istio-citadel-1.1.0 && \
```
Comment (brian-avery) [inline Dockerfile.citadel:18]: Do we still need the version here? I believe you have to manually request what build you want for copr unless you're building the latest.

File: Dockerfile.istio-cni [hunk 2]
```diff
@@ -13,13 +13,14 @@ LABEL maintainer="Istio Feedback <istio-feedback@redhat.com>"
...
-
-RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/istio.el8/repo/rhelbeta-8/group_maistra-istio.el8-rhelbeta-8.repo -o /etc/yum.repos.d/openshift-istio-istio-epel-8.repo && \
+RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/istio-1.1/repo/epel-8/group_maistra-istio-1.1-epel-8.repo -o /etc/yum.repos.d/maistra-1.1-epel-8.repo && \
     microdnf update -y && \
-    microdnf install -y jq hostname istio-cni-1.1.0 && \
+    microdnf install -y hostname istio-cni && \
     microdnf clean all
-WORKDIR /tmp/
+# jq is not available in ubi8 yum repo
+RUN curl -Ls https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /usr/bin/jq && chmod +x /usr/bin/jq
```
Comment (brian-avery) [inline Dockerfile.istio-cni:21]: it works.

File: artifacts/gather_istio [hunk 3]
```diff
@@ -68,21 +69,87 @@ function getCRDs() {
...
+      echo "Collecting config for pod ${podName}.${podNamespace}"
+
+      local logPath=${BASE_COLLECTION_PATH}/namespaces/${podNamespace}/pods/${podName}
+      mkdir -p ${logPath}
+
+      echo "Pilot config for pod ${podName}.${podNamespace} from pilot ${pilotName}.${controlPlaneNamespace}" 2>&1 1>${logPath}/pilotConfiguration
+      oc exec ${pilotName} -n ${controlPlaneNamespace} -c discovery -- bash -c "/usr/local/bin/pilot-discovery request GET /debug/config_dump?proxyID=${podName}.${podNamespace}" 2>&1 1>${logPath}/pilotConfiguration
+
+      echo "Envoy config for pod ${podName}.${podNamespace} from pilot ${pilotName}.${controlPlaneNamespace}" 2>&1 1>${logPath}/envoyConfiguration
+      oc exec -n ${podNamespace} ${podName} -c istio-proxy -- /usr/local/bin/pilot-agent request GET config_dump 2>&1 1>${logPath}/envoyConfiguration
```
Comment (brian-avery) [inline artifacts/gather_istio:125]: this filename should be different than the one above

**Review by brian-avery — APPROVED** [review]

File: Dockerfile.citadel (hunk 1 above)
Comment (jwendell) [reply Dockerfile.citadel:18]: We are pulling from an exclusive 1.1 yum repo, so, no need to specify the version here.

File: Dockerfile.istio-cni (hunk 2 above)
Comment (jwendell) [reply Dockerfile.istio-cni:21]: :)

**Review by brian-avery — APPROVED** [review]

**Review by knrc — APPROVED** [review]

**Review by dgn — APPROVED** [review]

**Pull request merged**